app/repositories/ArrivalMovementRepository.scala

```diff
@@ -64,12 +79,15 @@ class ArrivalMovementRepository @Inject()(mongo: ReactiveMongoApi)(implicit ec:
   private def collection: Future[JSONCollection] =
     mongo.database.map(_.collection[JSONCollection](collectionName))
 
-  def insert(arrival: Arrival): Future[Unit] =
+  def insert(arrival: Arrival): Future[Unit] = {
+    val updatedArrival = arrival copy (lastUpdated = LocalDateTime.now)
```

Comment:
Since you've defaulted the constructor on the `Arrival` model to be `LocalDateTime.now` is this necessary?

Reply:
@amaalali This is just extra protection and to make us more aware when we look into it

Reply:
In that case we can remove it, it's the same as the Arrival status field, which should be `Initialised` at this point but that isn't the responsibility of this layer to enforce